### PR TITLE
Move to a new VPC.

### PR DESCRIPTION
**Pull request by MancunianSam**
This was sharing the Jenkins VPC but the Jenkins VPC has been destroyed
so this needs its own one.

This has already been deployed.

### CODE REVIEW

File: terraform/modules/grafana/variables.tf [hunk 1]
```diff
@@ -20,6 +20,6 @@ variable "ip_allowlist" {
 
 variable "kms_key_id" {}
 
-variable "vpc_cidr_block" {}
-
 variable "vpc_id" {}
+
+variable "private_subnet_ids" {}
```
Comment (TomJKing) [inline terraform/modules/grafana/variables.tf]: Missing line break

**Review by TomJKing — COMMENTED** [review]
Looks good. One very minor comment.

File: terraform/modules/grafana/variables.tf (hunk 1 above)
Comment (MancunianSam) [reply terraform/modules/grafana/variables.tf]: Sorry, I didn't check this one. I've added in the line break.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**